include/quiche.h

```diff
@@ -270,6 +270,10 @@ enum quiche_shutdown {
     QUICHE_SHUTDOWN_WRITE = 1,
 };
 
+// Set the priority for a stream.
```

Comment:
```suggestion
// Sets the priority for a stream.
```
(to match the `Connection::stream_priority()` Rust docs)